Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/ethereum/trin"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-discv5 = { git = "https://github.com/sigp/discv5.git", branch = "master" }
+discv5 = "0.1.0-beta.13"
```

Comment:
I thought we decided by default to track the master branch instead of the latest official release?